ci/requirements-openvino-dev.txt

```diff
@@ -0,0 +1,155 @@
+# use update-requirements.py to update this file
+
+addict==2.4.0
+    # via -r ci/requirements-openvino-dev.in
+certifi==2021.10.8
+    # via requests
+charset-normalizer==2.0.8
+    # via requests
+click==8.0.3
+    # via nltk
+colorama==0.4.4
+    # via
+    #   click
+    #   tqdm
+cycler==0.11.0
+    # via matplotlib
+decorator==4.4.2
+    # via networkx
+defusedxml==0.7.1
+    # via -r ci/requirements-openvino-dev.in
+editdistance==0.6.0
+    # via -r ci/requirements-openvino-dev.in
+fast-ctc-decode==0.3.0
+    # via -r ci/requirements-openvino-dev.in
+future==0.18.2
+    # via hyperopt
+hyperopt==0.1.2
+    # via -r ci/requirements-openvino-dev.in
+idna==3.3
+    # via requests
+imageio==2.13.1
+    # via scikit-image
+importlib-metadata==4.8.2
+    # via click
+joblib==1.1.0
+    # via
+    #   nltk
+    #   scikit-learn
+jstyleson==0.0.2
+    # via -r ci/requirements-openvino-dev.in
+kiwisolver==1.3.1
+    # via matplotlib
+matplotlib==3.3.4
+    # via scikit-image
+networkx==2.5.1
+    # via
+    #   -r ci/requirements-openvino-dev.in
+    #   hyperopt
+    #   scikit-image
+nibabel==3.2.1
+    # via -r ci/requirements-openvino-dev.in
+nltk==3.6.5
+    # via -r ci/requirements-openvino-dev.in
+numpy==1.19.5
+    # via
+    #   -r ci/requirements-openvino-dev.in
+    #   hyperopt
+    #   imageio
+    #   matplotlib
+    #   nibabel
+    #   opencv-python
+    #   openvino
+    #   pandas
+    #   parasail
+    #   pywavelets
+    #   rawpy
+    #   scikit-image
+    #   scikit-learn
+    #   scipy
+    #   tifffile
+opencv-python==4.5.4.60
+    # via -r ci/requirements-openvino-dev.in
+openvino==2021.4.2
```

Comment:
probably, this requirement should be relaxed, otherwise we can not move downloader precommit on new openvino package until new openvino-dev version will be released